src/main.rs

```diff
@@ -192,6 +208,46 @@ struct CheckRelease {
     )]
     release_type: Option<ReleaseType>,
 
+    /// Use only the default and explicitly added features.
+    #[arg(
+        long,
+        help_heading = "Features",
+        conflicts_with = "only_explicit_features"
+    )]
+    default_features: bool,
+
+    /// Use no features except the explicitly mentioned ones.
+    #[arg(long, help_heading = "Features")]
+    only_explicit_features: bool,
+
+    /// Use the named features.
+    #[arg(long, value_name = "NAME", help_heading = "Features")]
+    features: Vec<String>,
+
+    /// Use the named features in the baseline version only.
+    #[arg(long, value_name = "NAME", help_heading = "Features")]
+    baseline_features: Vec<String>,
+
+    /// Use the named features in the current version only.
+    #[arg(long, value_name = "NAME", help_heading = "Features")]
+    current_features: Vec<String>,
+
+    /// Use all the features.
+    /// By default, features with names:
+    /// `unstable`, `nightly`, `bench`, `no_std` or starting with `__` are not used.
```

Comment:
To me this conveys the exact opposite of the intended meaning: it seems to say "when you use `--all-features`, by default we omit the following list of features anyway: `unstable`, `nightly`, ..."

Reply:
Okay, I see